src/client.ts

```diff
@@ -624,8 +624,17 @@ export default class Client implements ClientInterface {
 
   private async getServerVersion(): Promise<string> {
     let bundleGemfile;
-
-    if (fs.existsSync(path.join(this.workingFolder, ".ruby-lsp", "Gemfile"))) {
+    const customBundleGemfile: string = vscode.workspace
+      .getConfiguration("rubyLsp")
+      .get("bundleGemfile")!;
+
+    // If a custom Gemfile was configured outside of the project, use that. Otherwise, prefer our custom bundle over the
```

Comment:
> configured outside of the project

Is this at odds with using `. getConfiguration("rubyLsp")`?

Reply:
No, by outside the project we mean outside of the current working directory. Some users like to have a separate `Gemfile` only for development dependencies and we provide this configuration as a way to point to it.

For example, something like this
```shell
# Regular bundle
project/Gemfile

# Ruby LSP custom bundle
project/.ruby-lsp/Gemfile

# User configured custom bundle
~/some/other/path/outside/of/project/Gemfile
```